jib-core/src/main/java/com/google/cloud/tools/jib/api/DockerClient.java

```diff
@@ -69,4 +69,15 @@ void save(ImageReference imageReference, Path outputPath, Consumer<Long> written
    * @throws InterruptedException if the {@code docker inspect} process was interrupted
    */
   ImageDetails inspect(ImageReference imageReference) throws IOException, InterruptedException;
+
+  /**
+   * Gets the os and architecture of the local docker installation.
```

Comment:
This could be just docker info details, `os and architecture` are implementation details of `DockerInfoDetails`, and we could add more attributes later.